py-polars/polars/expr/datetime.py

```diff
@@ -1548,6 +1548,11 @@ def convert_time_zone(self, time_zone: str) -> Expr:
         time_zone
             Time zone for the `Datetime` expression.
 
+        Notes
+        -----
+        If converting from a time-zone-naive datetime, then conversion will happen
+        as if converting from UTC, regardless of your system's time zone.
```

Comment:
calling this out, as Python stdlib interprets `datetime` according to your system's default time zone

Reply:
Good to include in the notes :+1: converting from UTC seems sensible.